boards/arc/hsdk/hsdk_2cores.dts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019, Synopsys, Inc. All rights reserved.
```

Comment:
2020

Reply:
Also there seem to be no difference between 4- & 2-core versions of `.dts` thus it makes sense to move all in `hsdk.dtsi` and include it in both 4- & 2-core versions.